### PR TITLE
16722 - update NR affiliation modal

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:*
https://github.com/bcgov/entity/issues/16722

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2469-7dx7pg9r.web.app

File: auth-web/src/components/auth/manage-business/AddNameRequestFormOld.vue [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+
+.form__btns {
+  display: flex;
+  justify-content: flex-end;
+
+  .v-btn + .v-btn {
+    margin-left: 0.5rem;
+  }
+
+  .v-btn[data-test="cancel-button"],
```
Comment (seeker25) [inline auth-web/src/components/auth/manage-business/AddNameRequestFormOld.vue:238]: Probably shouldn't be binding by data-test - I think we should look for ID in this case

File: auth-web/src/components/auth/manage-business/EntityManagement.vue [hunk 2]
```diff
@@ -921,6 +981,10 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
   closeBusinessUnavailableDialog () {
     this.$refs.businessUnavailableDialog.close()
   }
+
+  get isEnableBusinessNrSearch (): boolean {
+    return LaunchDarklyService.getFlag(LDFlags.EnableBusinessNrSearch) || false
```
Comment (seeker25) [inline auth-web/src/components/auth/manage-business/EntityManagement.vue:986]: I don't think we need this? this whole branch is for this feature?

**Review by seeker25 — APPROVED** [review]
High level looks good to me

**Comment by seeker25** [comment]
I think we're going to have to change this for staff:

image

**Pull request merged**